frontend/src/pages/modelRegistry/screens/RegisterModel/utils.ts

```diff
@@ -33,6 +33,7 @@ export const registerModel = async (
       name: formData.modelName,
       description: formData.modelDescription,
       customProperties: {},
+      author,
```

Comment:
This line will need to be:
```
owner: author,
```
because the property we're adding on RegisteredModel is `owner`, but we can leave the function argument called `author` because we're using the same value for the `author` of the ModelVersion and ModelArtifact in the `registerVersion` call below.

Alternatively, if you want you can change this `registerModel` function so it takes an `owner` instead of `author`, then this line is just `owner,` and then when we call `registerVersion()` below you can pass `owner` instead of `author`.

I feel like I'm talking in circles, hopefully this makes sense.